source/_components/netdata.markdown

```diff
@@ -49,15 +49,20 @@ To add this platform to your installation, add the following to your `configurat
 
 ```yaml
 # Example configuration.yaml entry
-sensor:
   - platform: netdata
-    resources:
-      load:
+    resources:    
+      system_load:
         data_group: system.load
         element: load15
-      cpu:
-        data_group: system.cpu
-        element: system
+        icon: mdi:chip
```

Comment:
Only required items in the first example

Reply:
removed non required data from the example